src/panels/config/zha/zha-config-dashboard.ts

```diff
@@ -0,0 +1,175 @@
+import {
+  LitElement,
+  TemplateResult,
+  html,
+  CSSResultArray,
+  css,
+  customElement,
+  property,
+  PropertyValues,
+} from "lit-element";
+import "@polymer/paper-item/paper-item-body";
+import "@polymer/paper-item/paper-item";
+import "../../../components/ha-card";
+import "../../../components/ha-icon-next";
+import "../../../layouts/hass-subpage";
+import "../ha-config-section";
+
+import { haStyle } from "../../../resources/styles";
+import { HomeAssistant, Route } from "../../../types";
+import { fetchDevices, ZHADevice } from "../../../data/zha";
+import { sortZHADevices, formatAsPaddedHex } from "./functions";
+import memoizeOne from "memoize-one";
+import {
+  DataTableColumnContainer,
+  RowClickedEvent,
+} from "../../../components/data-table/ha-data-table";
+import { navigate } from "../../../common/navigate";
+
+export interface DeviceRowData extends ZHADevice {
+  device?: DeviceRowData;
+}
+
+@customElement("zha-config-dashboard")
+class ZHAConfigDashboard extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public route!: Route;
+  @property() public narrow!: boolean;
+  @property() public isWide!: boolean;
+  @property() private _devices: ZHADevice[] = [];
+  private pages: string[] = ["add", "groups"];
+  private _firstUpdatedCalled: boolean = false;
+
+  private _memoizeDevices = memoizeOne((devices: ZHADevice[]) => {
+    let outputDevices: DeviceRowData[] = devices;
+
+    outputDevices = outputDevices.map((device) => {
+      return {
+        ...device,
+        name: device.user_given_name ? device.user_given_name : device.name,
+        nwk: formatAsPaddedHex(device.nwk),
+        id: device.ieee,
```

Comment:
You don't need to add `id` here you just specify that `ieee` is the unique id on `ha-data-table`

Reply:
Now the data is in it twice, both as `id` and as `ieee`

Reply:
See: https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/panels/lovelace/editor/unused-entities/hui-unused-entities.ts#L157

Reply:
noted! thanks.